transmogrifier/sources/json/aardvark.py

```diff
@@ -126,7 +126,7 @@ def get_optional_fields(self, source_record: dict) -> dict | None:
         fields["alternate_titles"] = self.get_alternate_titles(source_record) or None
 
         # content_type
-        fields["content_type"] = ["Geospatial data"]
+        fields["content_type"] = source_record.get("gbl_resourceType_sm")
```

Comment:
Ignoring the comment above about whether this be _allowed_ remain `None` or a potentially empty list, I think it's a good sign that this change was so slight.  Feels like the transformer for Aardvark records is flexible in a good way.